test/unit/controllers/ConsignmentReferencesControllerSpec.scala

```diff
@@ -90,61 +90,70 @@ class ConsignmentReferencesControllerSpec extends ControllerSpec with OptionValu
       }
     }
 
-    "return BadRequest for incorrect form" in {
+    "return 400 (BAD_REQUEST)" when {
 
-      givenAUserOnTheArrivalJourney()
+      "form is incorrect" in {
```

Comment:
We don't have `GET` in any test description

Reply:
I meant, that method name is mentioned in tests for "display", but very rarely for submit. It would be more explanatory to know that for. e.g. you expect 400 on submit, not display method call.